col doesn't like the bullet character used in this manpage, so replace it with
a simple indent.
diff --git a/doc/apctest.8 b/doc/apctest.8
--- a/doc/apctest.8
+++ b/doc/apctest.8
@@ -28,7 +28,7 @@
 IMPORTANT: Before running
 .Nm
 .Pp
-.Bl -bullet
+.Bl -hang
 .It
 Check that the UPSCABLE, UPSTYPE and DEVICE configuration directives in the
 .Xr apcupsd.conf 5
diff --git a/doc/apcupsd.conf.5 b/doc/apcupsd.conf.5
--- a/doc/apcupsd.conf.5
+++ b/doc/apcupsd.conf.5
@@ -43,7 +43,7 @@
 directive blank and apcupsd will figure out where the device is 
 located. 
 .Pp
-.Bl -bullet -compact
+.Bl -hang -compact
 .It 
 apcsmart
 : /dev/tty** (serial connection)
